x-pack/plugins/actions/server/builtin_action_types/resilient/case_schema.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const MappingActionType = schema.oneOf([
+  schema.literal('nothing'),
```

Comment:
The same comment as for Jira PR: _I think, this files `case_schema.ts` and `case_types.ts` make sense to move to a `case` folder - to avoid a code duplication all over the Case related action types and in the future it will be easier to find and clean it for all those types when the logic change._